packages/blockchain-wallet-v4-frontend/src/modals/BuySell/EnterAmount/Checkout/index.tsx

```diff
@@ -104,8 +103,6 @@ const Checkout = (props: Props) => {
         pair: props.pair,
```

Comment:
@milan-bc, it would be interesting to see if we can further simplify logic between Line 81 and 95 in this file.